main.go

```diff
@@ -390,7 +389,7 @@ func main() {
 		switch configs.Track {
 		case betaTrackName:
 			possibleTrackNamesToUpdate = []string{alphaTrackName}
-		case rolloutTrackName, productionTrackName:
+		case productionTrackName:
```

Comment:
Do not know if this 'Deactivating blocking apk versions' is still necessary or maybe the internal track needs to be added here also.